spi/common/http-spi/src/main/java/org/eclipse/edc/spi/http/FallbackFactories.java

```diff
@@ -67,4 +70,26 @@ static FallbackFactory retryWhenStatusIsNot(int status) {
                     .build();
         };
     }
+
+    /**
+     * Verifies that the response has a specific statuses, otherwise it should be retried
+     *
+     * @return the {@link FallbackFactory}
+     */
+    static FallbackFactory retryWhenStatusIsNotIn(int... status) {
```

Comment:
At this point `retryWhenStatusIsNot` could be inlined and all its callers could call this new method.